parachain/pallets/inbound-queue/src/mock.rs

```diff
@@ -21,8 +21,8 @@ use sp_runtime::{
 	BuildStorage, FixedU128, MultiSignature,
 };
 use sp_std::convert::From;
-use xcm::v3::{prelude::*, MultiAssets, SendXcm};
-use xcm_executor::Assets;
+use xcm::v4::{prelude::*, SendXcm};
```

Comment:
Seems the explicit `v4` is unnecessary here, it's by default exposed in `xcm::prelude`

Reply:
Good point! Changed in https://github.com/Snowfork/snowbridge/pull/1123/commits/b8515e6155e4fedd2c7842c6fcd84856c3a197fc